libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -54,17 +54,20 @@ struct PerViewUib { // NOLINT(cppcoreguidelines-pro-type-member-init)
 
     math::float4 resolution; // viewport width, height, 1/width, 1/height
 
+    // camera forward in view space (when camera_at_origin is enabled).
+    math::float3 cameraForward;
```

Comment:
No it doesn't. Good that you point out, because I was about to delete the comment altogether (the whole 'view space' terminology is confusing here and above `cameraPosition` too). Comment deleted in : 5e900c366faaa449afa76c44196b49b0389a1a74